src/SldStyleParser.ts

```diff
@@ -1615,62 +1438,242 @@ export class SldStyleParser implements StyleParser<string> {
   }
 
   /**
-   * Get the Label from a TextSymbolizer
+   * Get the SLD Object (readable with xml2js) from an GeoStyler-Style MarkSymbolizer.
+   *
+   * @param markSymbolizer A GeoStyler-Style MarkSymbolizer.
+   * @return The object representation of a SLD PointSymbolizer with a
+   * Mark (readable with xml2js)
    */
-  getSldLabelFromTextSymbolizer = (template: string): [any] => {
-    const openingBraces = '{{';
-    const closingBraces = '}}';
+  getSldPointSymbolizerFromMarkSymbolizer(markSymbolizer: GsMarkSymbolizer): any {
+    const WellKnownName = this.getTagName('WellKnownName');
+    const CssParameter = this.getTagName('CssParameter');
+    const Fill = this.getTagName('Fill');
+    const Mark = this.getTagName('Mark');
+    const Stroke = this.getTagName('Stroke');
+    const Opacity = this.getTagName('Opacity');
+    const Rotation = this.getTagName('Rotation');
+    const Size = this.getTagName('Size');
+    const Graphic = this.getTagName('Graphic');
 
-    const tokens = [];
-    let templateReducer = template;
+    const isFontSymbol = WELLKNOWNNAME_TTF_REGEXP.test(markSymbolizer.wellKnownName);
+    const mark: any[] = [{
+      [WellKnownName]: [{
+        '#text': isFontSymbol ? markSymbolizer.wellKnownName : markSymbolizer.wellKnownName.toLowerCase()
+      }]
+    }];
 
-    while(templateReducer.length) {
-      let tmpTemplateReducer = templateReducer;
-      let tmpPreTemplateLiteral;
-      const openingBracesIdx = tmpTemplateReducer.indexOf(openingBraces);
-      if (openingBracesIdx === -1) {
-        tokens.push({
-          'ogc:Literal': templateReducer
+    if (markSymbolizer.color || markSymbolizer.fillOpacity) {
+      const fillCssParamaters = [];
+      if (markSymbolizer.color) {
+        // TODO: parse GeoStylerFunctions
```

Comment:
Secnd occurence, should probably be turned into an issue?

Reply:
https://github.com/geostyler/geostyler-sld-parser/issues/697